integration/image/save_test.go

```diff
@@ -27,7 +27,6 @@ import (
 	"github.com/opencontainers/go-digest"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"gotest.tools/v3/assert"
-	"gotest.tools/v3/assert/cmp"
 	is "gotest.tools/v3/assert/cmp"
```

Comment:
Still need to look what linter we're missing that would flag the duplicate imports.